spec/actions/pkg_spec.cr

```diff
@@ -61,36 +56,36 @@ module DppmRestApi::Actions::Pkg
   describe DppmRestApi::Actions::Pkg do
     describe "list built packages" do
       it "responds with 401 Forbidden" do
-        get fmt_route nil
+        get fmt_route '/' + DPPM::Prefix.default_source_name
```

Comment:
What is this?

Reply:
We get all source packages from the default source.

Reply:
(Not implemented yet)

Reply:
We get all built packages from a given source, here the default one.